Unable to build on aarch64

I've been unable to build tmux-top successfully on aarch64 (Termux):

```bash
uname -a                                                                     
Linux localhost 4.9.200-perf-g44bd5bc #1 SMP PREEMPT Sun Nov 24 19:53:16 UTC 2019 aarch64 Android
go get github.com/TomasTomecek/tmux-top/cmd/tmux-top                       
# github.com/TomasTomecek/tmux-top/cmd/tmux-top
dev/golang/src/github.com/TomasTomecek/tmux-top/cmd/tmux-top/cli.go:115:4: unknown field 'ShortName' in 
struct literal of type cli.Command
dev/golang/src/github.com/TomasTomecek/tmux-top/cmd/tmux-top/cli.go:117:4: cannot use print_net (type fu
nc(*cli.Context)) as type cli.ActionFunc in field value
dev/golang/src/github.com/TomasTomecek/tmux-top/cmd/tmux-top/cli.go:121:4: unknown field 'ShortName' in 
struct literal of type cli.Command
dev/golang/src/github.com/TomasTomecek/tmux-top/cmd/tmux-top/cli.go:123:4: cannot use print_mem (type fu
nc(*cli.Context)) as type cli.ActionFunc in field value
dev/golang/src/github.com/TomasTomecek/tmux-top/cmd/tmux-top/cli.go:127:4: unknown field 'ShortName' in 
struct literal of type cli.Command
dev/golang/src/github.com/TomasTomecek/tmux-top/cmd/tmux-top/cli.go:129:4: cannot use print_load (type f
unc(*cli.Context)) as type cli.ActionFunc in field value
dev/golang/src/github.com/TomasTomecek/tmux-top/cmd/tmux-top/cli.go:133:4: unknown field 'ShortName' in 
struct literal of type cli.Command
dev/golang/src/github.com/TomasTomecek/tmux-top/cmd/tmux-top/cli.go:135:4: cannot use print_io (type fun
c(*cli.Context)) as type cli.ActionFunc in field value
dev/golang/src/github.com/TomasTomecek/tmux-top/cmd/tmux-top/cli.go:139:4: unknown field 'ShortName' in 
struct literal of type cli.Command
dev/golang/src/github.com/TomasTomecek/tmux-top/cmd/tmux-top/cli.go:141:4: cannot use print_sens (type f
unc(*cli.Context)) as type cli.ActionFunc in field value
dev/golang/src/github.com/TomasTomecek/tmux-top/cmd/tmux-top/cli.go:141:4: too many errors
```

Cross compiling for ARM works though (and produces a usable binary)
```
CC=arm-linux-gnueabihf-gcc GOOS=linux GOARCH=arm GOARM=7 CGO_ENABLED=0 go build -a -ldflags '-extldflags "-static"' -v -o tmux-top_linux_armv8_static -buildmode=default ./cmd/tmux-top
```

--- a/go.mod
+++ b/go.mod
@@ -1,0 +1,5 @@
+module github.com/TomasTomecek/tmux-top
+
+go 1.13
+
+require github.com/urfave/cli v1.22.2

